src/Sulu/Component/Content/Tests/Unit/Document/Subscriber/FallbackLocalizationSubscriberTest.php

```diff
@@ -132,14 +128,9 @@ public function testNoWebspaceDoNotLoadGhostContent()
     {
         $this->inspector->getWebspace($this->document->reveal())->willReturn(null);
         $this->inspector->getLocales($this->document)->willReturn(['de', 'fr']);
-        $this->hydrateEvent->setLocale('de')->shouldBeCalled();
+        $this->hydrateEvent->setLocale('de')->shouldNotBeCalled();
```

Comment:
no because if you dont load ghost content it should not be changed for other event subscribers.